pyvcloud/vcd/org.py

```diff
@@ -638,3 +642,134 @@ def update_org(self, org_name, is_enabled=None):
+        :param cpu_units (str): The cpu units compute capacity allocated to this vDC. One of MHz or GHz
+        :param cpu_allocated (int): Capacity that is committed to be available.
+        :param cpu_limit (int): Capacity limit relative to the value specified for Allocation.
+        :param mem_units (str): The memory units compute capacity allocated to this vDC. One of MB or GB.
+        :param mem_allocated (int): Memory capacity that is committed to be available.
+        :param mem_limit (int): Memory capacity limit relative to the value specified for Allocation.
+        :param nic_quota (int): Maximum number of virtual NICs allowed in this vDC. Defaults to 0, which specifies an unlimited number.
+        :param network_quota (int): Maximum number of network objects that can be deployed in this vDC. Defaults to 0, which means no networks can be deployed.
+        :param vm_quota (int): The maximum number of VMs that can be created in this vDC. Defaults to 0, which specifies an unlimited number.
+        :param storage_profiles: List of provider vDC storage profiles to add to this vDC.
+            Each item is a dictionary that should include the following elements:
+                name: (string) name of the PVDC storage profile.
+                enabled: (bool) True if the storage profile is enabled for this vDC.
+                units: (string) Units used to define limit. One of MB or GB.
+                limit: (int) Max number of units allocated for this storage profile.
+                default: (bool) True if this is default storage profile for this vDC.
+        :param resource_guaranteed_memory (float): Percentage of allocated CPU resources guaranteed to vApps deployed in this vDC.
+            Value defaults to 1.0 if the element is empty.
+        :param resource_guaranteed_cpu (float): Percentage of allocated memory resources guaranteed to vApps deployed in this vDC.
+            Value defaults to 1.0 if the element is empty.
+        :param vcpu_in_mhz (int): Specifies the clock frequency, in Megahertz, for any virtual CPU that is allocated to a VM.
+        :param is_thin_provision (bool): Boolean to request thin provisioning.
+        :param network_pool_name (str): Reference to a network pool in the Provider vDC.
+        :param uses_fast_provisioning (bool): Boolean to request fast provisioning.
+        :param over_commit_allowed (bool): Set to false to disallow creation of the VDC if the AllocationModel is AllocationPool or ReservationPool
+            and the ComputeCapacity you specified is greater than what the backing Provider VDC can supply. Defaults to true if empty or missing.
+        :param vm_discovery_enabled (bool): True if discovery of vCenter VMs is enabled for resource pools backing this vDC.
+        :param is_enabled (bool): True if this vDC is enabled for use by the organization vDCs.
```

Comment:
is_enabled : "for use by the organization vDCs". Did you mean users?

Reply:
yes, will update.